sdk/python/foundation-models/huggingface/inference/text-to-image/text-to-image-online-endpoint.ipynb

```diff
@@ -0,0 +1,297 @@
+    "from azure.ai.ml.entities import ManagedOnlineEndpoint, ManagedOnlineDeployment\n",
+    "\n",
+    "# Endpoint names need to be unique in a region, hence using timestamp to create unique endpoint name\n",
+    "timestamp = int(time.time())\n",
+    "online_endpoint_name = \"text-to-image-\" + str(timestamp)\n",
+    "# Create an online endpoint\n",
+    "endpoint = ManagedOnlineEndpoint(\n",
+    "    name=online_endpoint_name,\n",
+    "    description=\"Online endpoint for \" + diffusion_models.name + \", for text-to-image task\",\n",
+    "    auth_mode=\"key\",\n",
+    ")\n",
+    "workspace_ml_client.begin_create_or_update(endpoint).wait()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.ai.ml.entities import OnlineRequestSettings, ProbeSettings\n",
+    "\n",
+    "deployment_name = \"text-to-image-mlflow-deploy\"\n",
+    "\n",
+    "# Create a deployment\n",
+    "demo_deployment = ManagedOnlineDeployment(\n",
+    "    name=deployment_name,\n",
+    "    endpoint_name=online_endpoint_name,\n",
+    "    model=diffusion_models.id,\n",
+    "    instance_type=\"Standard_NC6s_v3\", # Use GPU instance type like Standard_NC6s_v3 or above\n",
+    "    instance_count=1,\n",
+    "    request_settings=OnlineRequestSettings(\n",
+    "        max_concurrent_requests_per_instance=1,\n",
+    "        request_timeout_ms=90000,\n",
+    "        max_queue_wait_ms=500\n",
+    "    ),\n",
+    "    liveness_probe=ProbeSettings(\n",
+    "        failure_threshold=49,\n",
+    "        success_threshold=1,\n",
+    "        timeout=299,\n",
+    "        period=180,\n",
+    "        initial_delay=180,\n",
+    "    ),\n",
+    "    readiness_probe=ProbeSettings(\n",
+    "        failure_threshold=10,\n",
+    "        success_threshold=1,\n",
+    "        timeout=10,\n",
+    "        period=10,\n",
+    "        initial_delay=10,\n",
+    "    ),\n",
+    ")\n",
+    "workspace_ml_client.online_deployments.begin_create_or_update(demo_deployment).wait()\n",
+    "endpoint.traffic = {deployment_name: 100}\n",
+    "workspace_ml_client.begin_create_or_update(endpoint).result()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4. Test the endpoint\n",
+    "\n",
+    "We will submit request to online endpoint with some sample text prompts for image generation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "demo_deployment = workspace_ml_client.online_deployments.get(\n",
+    "    name=deployment_name,\n",
+    "    endpoint_name=online_endpoint_name,\n",
+    ")\n",
+    "\n",
+    "# Get the details for online endpoint\n",
+    "endpoint = workspace_ml_client.online_endpoints.get(name=online_endpoint_name)\n",
+    "\n",
+    "# Existing traffic details\n",
+    "print(endpoint.traffic)\n",
```

Comment:
Why we need to print this details?
demo_deployment obj is already available in above cell.

I think, we should keep the notebook simple.

Reply:
I am following foundation-models notebooks, lets change it there as well.